<launch>
  <arg name="rate" default="10" />
  <arg name="namespace" default="$(env UAV_NAMESPACE)" />
  <arg name="num_of_batt" default="1" />
  <arg name="batt_time" default="[3, 3]" />
  <arg name="length_x" default="1" />
  <arg name="length_y" default="1" />
  <arg name="spacing" default="1" />
  <arg name="base_x" default="1" />
  <arg name="base_y" default="1" />
  <arg name="base_z" default="1" />
  <arg name="altitude" default="1" />

  <group ns="$(arg namespace)">
    <node name="path_optimization" pkg="path_planning_optimization" type="path_optimization.py" output="screen">
      <param name="rate" value="20" />
      <param name="num_of_batt" value="2" />
      <rosparam param="batt_time" subst_value="True">$(arg batt_time)</rosparam>
      <param name="length_x" value="10" />
      <param name="length_y" value="10" />
      <param name="spacing" value="1" />
      <param name="base_x" value="0" />
      <param name="base_y" value="0" />
      <param name="base_z" value="1" />
      <param name="altitude" value="5" />
    </node>
  </group>
</launch>
